### ISSUE TITLE
RoBERTa numbers without MNLI?

### ISSUE THREAD

**Issue by sleepinyourhat** [pytorch/fairseq#1166]
The RoBERTa paper suggests that the test set results for a few tasks (MRPC, RTE, STS) involved three-stage MLM->MNLI->[task] training, and I'm curious if there are any results available that compare RoBERTa with and without MNLI and that setup.

I'm asking here because the development set results on the RoBERTa readme here are a bit ambiguous/confusing: It seems like the no-MNLI validation results in the readme precisely match the validation results in the paper, which presumably used MNLI.

**Comment by ngoyal2707** [comment]
Hey Sam, sorry if it's unclear but the validation results reported in paper and Readme are `MLM -> [task]`. 

Lemme find validation results for MLM -> MNLI -> [task] and report back here

**Comment by ngoyal2707** [comment]
Validation results for MRPC, RTE and STS-B in `MLM->MNLI->task` settings are `91.42, 90.97, 92.63` in order (median of 5 seeds).
